home/forms.py

```diff
@@ -3,6 +3,8 @@
 
 
 class QuestionForm(forms.ModelForm):
+    tags_ = forms.CharField(max_length=110, min_length=0, required=False)
+
     class Meta:
         model = Question
-        fields = ('title', 'subject', 'sub_subject', 'grade', 'book', 'book_page', 'content')
+        fields = ('title', 'subject', 'sub_subject', 'grade', 'book', 'book_page', 'tags_', 'content')
```

Comment:
is it a typo? "tags_" ?

Reply:
No. Question model has a field tags, so the '_' was added to separate the model field and the form field.